java/ql/src/utils/stub-generator/Stubs.qll

```diff
@@ -391,6 +402,24 @@ private string stubMember(Member m) {
   )
 }
 
+private string stubAnnotation(Annotation a) {
+  if exists(a.getAValue())
+  then result = a.toString() + "(" + concat(stubAnnotationValue(a.getAValue()), ",") + ")"
```

Comment:
Instead of `a.toString()`, would it make sense to explicitly use `a.getType().getName()`? `toString()` does not make any guarantees about its result.

Reply:
Done in 495ce95f3856ca9d5416f312abb96624ff306b1e.